usaspending_api/routers/broker.py

```diff
@@ -0,0 +1,12 @@
+class BrokerRouter(object):
```

Comment:
Technically the behavior of this router is already encapsulated in the other two router (DefaultOnlyRouter and ReadReplicaRouter).  I am impartial as to whether or not we want to keep this in addition to those, but it does seem unnecessary.

Reply:
Yeah, i see that this new code
```
    def allow_migrate(self, db, app_label, model_name=None, **hints):
        """ Migrations should only run in USAspending against the writable database. """
        return db == self.writable_database
``` 
basically covers what I was trying to do, and what was missing from our route impls.

Reply:
Done